modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -1476,11 +1476,6 @@ private void resetDirtyPages() {
         public boolean tryToRemovePage(FullPageId fullPageId, long absPtr) throws IgniteInternalCheckedException {
             assert writeLock().isHeldByCurrentThread();
 
-            // Do not evict group meta pages.
```

Comment:
Because it was stored in the index partition, and now it will not be. It is not entirely clear whether this page will be needed in the future, in 2.0 information for snapshots was stored there before metastorage, now I think this information will be located in the configuration.

Reply:
ok!